### ISSUE TITLE
Include new filters for WordPress 4.4

### ISSUE THREAD

**Issue by DrizzlyOwl** [roikles/Flexbones#41]
Suggest adding the following into filters.php to help in reducing the amount of chaff in `wp_head()`

```php
// Disable REST API 
add_filter('rest_enabled', '_return_false');
add_filter('rest_jsonp_enabled', '_return_false');

// Remove oembed links & REST urls
remove_action( 'wp_head', 'rest_output_link_wp_head', 10 );
remove_action( 'wp_head', 'wp_oembed_add_discovery_links', 10 );

// Deregister wp-embed scripts if not in use
function deregister_scripts(){
    wp_deregister_script( 'wp-embed' );
}
add_action( 'wp_footer', 'deregister_scripts' );
```

**Comment by roikles** [comment]
With regards to wp-embeds I need to do some testing because I like the simplicity of oEmbed so I would rather find a way to add it to the main scripts.min.js file ideally. More testing is needed.

What do the other filters and actions do?

**Comment by DrizzlyOwl** [comment]
Updated the OP with comments. 

I've also seen this thread on S.O. which could be handy! http://wordpress.stackexchange.com/a/212472

**Comment by roikles** [comment]
Im going to make these toggleable from the theme customizer.

**Comment by roikles** [comment]
Fixed: added this to admin theme customizer d62ad3f7ad0614c8a99cec14fb5f2b871db259e4
